julia_src/reopt_model.jl

```diff
@@ -1410,6 +1611,67 @@ function reopt_results(m, p, r::Dict)
 	return r
 end
 
+function add_re_emissions_results(m, p, r::Dict)
+
+	r["pwf_om"] = round(value(p.pwf_om),digits=4)
+
+	# renewable elec
+	r["annual_re_elec_kwh"] = round(value(m[:AnnualREEleckWh]), digits=2)
+	m[:AnnualREElecPercent] = @expression(m, m[:AnnualREEleckWh]/(sum(p.ElecLoad[ts] for ts in p.TimeStep)))
+	r["annual_re_elec_percent"] = round(value(m[:AnnualREElecPercent]), digits=6)
+
+	# renewable heat
+	if !isempty(p.HeatingTechs)
+		add_re_heat_calcs(m,p)
+		r["annual_re_heat_mmbtu"] = round(value(m[:AnnualREHeatMMBTU]), digits=2)
+		r["annual_heat_mmbtu"] = round(value(m[:AnnualHeatMMBTU]), digits=2)
+		r["annual_re_heat_percent"] = round(value(m[:AnnualREHeatMMBTU])/value(m[:AnnualHeatMMBTU]),digits=6)
+	else
+		r["annual_re_heat_mmbtu"] = 0.0
+		r["annual_re_heat_percent"] = 0.0
+	end
+
+	# BAU preprocessed emissions (used for emissions reduction target calculations; output for testing purposes)
+	r["preprocessed_BAU_year_one_emissions_tCO2"] = round(value(p.BAUYr1Emissions_CO2)/2204.62,digits=2)
+	r["preprocessed_BAU_lifecycle_emissions_tCO2"] = round(value(m[:Lifecycle_Emissions_Lbs_CO2_BAU]/2204.62),digits=2) # no annual decrease for on-site fuel burn
+
+	# Year 1 Emissions results at Site level
+	r["year_one_emissions_tCO2"] = round(value(m[:EmissionsYr1_Total_LbsCO2]/2204.62), digits=2)
+	r["year_one_emissions_tNOx"] = round(value(m[:EmissionsYr1_Total_LbsNOx]/2204.62), digits=2)
+	r["year_one_emissions_tSO2"] = round(value(m[:EmissionsYr1_Total_LbsSO2]/2204.62), digits=2)
+	r["year_one_emissions_PM25"] = round(value(m[:EmissionsYr1_Total_LbsPM25]/2204.62), digits=2)
```

Comment:
missing "t" before PM25

Reply:
addressed in [9d4c8e3](https://github.com/NREL/REopt_Lite_API/pull/274/commits/9d4c8e3f5c7091a09cb6be5932bd695c65a9a949). Thanks!